lightkurve/lightcurve.py

```diff
@@ -1181,6 +1181,12 @@ def query_solar_system_objects(self, cadence_mask='outliers', radius=None,
             DataFrame object which lists the Solar System objects in frames
             that were identified to contain SSOs.  Returns `None` if no objects
             were found.
+
+        Examples
+        --------
+        Find if there are SSOs affecting the lightcurve for the given time frame:
+
+            >>> df_sso = lc.query_solar_system_objects(cadence_mask=np.logical_and(lc.time >= 2014.1, lc.time <= 2014.9))
```

Comment:
By default our continuous testing will try to evaluate this example, which in this case is failing because `lc` is not defined.  We either have to expand the example a little, or otherwise add `# doctest: +SKIP` behind the line to make the testing skip the line:

```suggestion
            >>> df_sso = lc.query_solar_system_objects(cadence_mask=(lc.time.value >= 2014.1) & (lc.time.value <= 2014.9))  # doctest: +SKIP
```

In the suggested change, I also changed `lc.time` into `lc.time.value` which is necessary now that `lc.time` is an AstroPy Time object as of Lightkurve v2.x.

Reply:
fixed.